modules/36_buildings/simple/datainput.gms

```diff
@@ -35,15 +35,25 @@ pm_cesdata_sigma(ttot,"enhgab")$ (ttot.val eq 2040) = 3;
 
 
 *** floor space demand for reporting
+* RH, Jan 2023: if clauses to maintain backwards compatibility with old input data for a while
 Parameter
+$call "awk -F ',' '$0 !~ /^\*/ { exit NF }' ./modules/36_buildings/simple/input/p36_floorspace_scen.cs4r"
```

Comment:
This does add an `awk` dependency to REMIND.  I would defer to RSE(@LaviniaBaumstark, @mikapfl) to give a thumbs up or down on that.  Is `buldings/simple` the realisation that is light-weight and that people are going to use for testing their own developments on the local Windows machine?
Three available courses of action:
- We don't care.
- Tell people how to install `awk` under Windows (I guess something like [this](https://github.com/pik-piam/discussions/discussions/49)).
- Use the [alternate formulation for two different input files](https://mattermost.pik-potsdam.de/rd3/pl/bq8xaqkpuj8szbhiaizw6bbxga) (depending on [what happens to mrremind#352](https://github.com/pik-piam/mrremind/pull/352#pullrequestreview-1270624336)).  That is based only on GAMS and has the added benefit of being "more backwards compatible" (on the `mrremind` end).

Reply:
I don't feel I can say much about those 3 options. But I do know that `buildings/simple` is the most basic one that will be used for those reasons.

Reply:
More importantly: That line needs an explanatory comment, I have no idea what that awk thing is doing.

Reply:
```diff
+!! return the number of colons on the first line not starting with "*" in p36_floorspace_scen.cs4r as errorLevel
 $call "awk -F ',' '$0 !~ /^\*/ { exit NF }' ./modules/36_buildings/simple/input/p36_floorspace_scen.cs4r"
```

Reply:
You're right, @mikapfl. I added a respective comment.

Reply:
I think option 3 is the best solution

Reply:
I now went with option 3. `mrremind` now creates a new file `f36_floorspace.cs4r` that has the extra dimension. This removes the `awk` dependency but wont allow to use old input data with a new REMIND version. However, this does not seem to be a usual use case as @LaviniaBaumstark told me.

Reply:
yes, we sometimes change input data such that you need a specific model version. That is o.k. sounds good for me. Do you have generated new input date including those changes and tested REMIND with it?

Reply:
I did successful test a while back but no current ones. I wanted to be lazy and wait for the next input data update, test this REMIND version with it and then push it.

Reply:
Yes, that's correct.